### PR TITLE
Build with TF 2.11.0

### PR DESCRIPTION
**Pull request by bhack**
Build the release with TF 2.11.0

### CODE REVIEW

**Comment by bhack** [comment]
@chenmoneygithub  Can you check the optimizer 2.11 errors?

**Comment by chenmoneygithub** [comment]
@bhack I made #2783 to fix optimizer failures. The error log is too long to read, so I might miss some cases.

**Comment by bhack** [comment]
> We need to remove `GroupNormalization` layer as it has been incorporated in the core now

Yes but I think that we need to give the same deprecation warming. Can you send a PR against `build_2.11.0` branch?

**Comment by AakashKumarNain** [comment]
> > We need to remove `GroupNormalization` layer as it has been incorporated in the core now
> 
> Yes but I think that we need to give the same deprecation warning like other "duplicated" components. Can you send a PR against `build_2.11.0` branch?

I can do it over the weekend. Got a lot on my plate right now. Will that work?

**Comment by bhack** [comment]
> I can do it over the weekend. Got a lot on my plate right now. Will that work?

Yes it is ok as we are waiting also for the Macos release https://pypi.org/project/tensorflow-macos/#history

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -101,20 +101,20 @@ jobs:
       matrix:
         os: ['macOS', 'Windows', 'Linux']
         py-version: ['3.7', '3.8', '3.9', '3.10']
-        tf-version: ['2.10.0']
+        tf-version: ['2.9.3', '2.10.1', '2.11.0']
```
Comment (seanpmorgan) [inline .github/workflows/release.yml:104]: We only publish wheels that were built with the most recent TF version.

Comment (bhack) [reply .github/workflows/release.yml]: Yes we had some drift with https://github.com/tensorflow/addons/blob/master/tools/update_release_version.sh.
I think that we could fix that utility that I've contributed as it is still useful to quickly prepare a release version PR for the "repetitive" tasks.

**Comment by apivovarov** [comment]
TF 2.11 still not supported because `EXCLUSIVE_MAX_TF_VERSION = "2.11.0"` in https://github.com/tensorflow/addons/blob/master/tensorflow_addons/version.py

**Pull request merged**